src/client/SConscript

```diff
@@ -9,6 +9,7 @@ def scons():
     SConscript('dfs/SConscript')
     SConscript('dfuse/SConscript')
     SConscript('pydaos/SConscript')
+    SConscript('scm_estimator/SConscript')
```

Comment:
storage_estimator ?  I think we need a better name

Reply:
I changed the name to storage_estimator